src/OpenTK.Windowing.Desktop/GameWindow.cs

```diff
@@ -419,5 +432,35 @@ public void ResetTimeSinceLastUpdate()
         {
             _watchUpdate.Restart();
         }
+
+        private void Win32_OnModalSizeMoveBegin()
+        {
+            if (_win32SuspendTimerOnDrag)
+            {
+                _watchUpdate.Stop();
+            }
+        }
+
+        private void Win32_OnModalSizeMoveEnd()
+        {
+            if (_win32SuspendTimerOnDrag)
+            {
+                _watchUpdate.Restart();
+            }
+        }
```

Comment:
If we never register the wndproc hook if we don't have this turned on we don't need to do the if check here.

Reply:
I'll talk about this one in Discord. I've run some tests and a few other messages do fire during modal size/move, although I haven't backtracked to their WM definitions yet. I'm wondering if that might be used to drive the render loop -- optionally, which means we'd want to handle multiple flags here. I'll get the definitions then ping you in chat. But by the numbers, the first set is during a brief drag, the second set is during a brief resize:

```
--- (moving)
msg 0x02a2      count 1
msg 0x0216      count 352
msg 0x0046      count 352
msg 0x0024      count 352
msg 0x0047      count 352
msg 0x0003      count 352
msg 0x000d      count 1

--- (resizing)
msg 0x02a2      count 1
msg 0x0214      count 36
msg 0x0046      count 36
msg 0x0024      count 36
msg 0x0083      count 36
msg 0x0085      count 36
msg 0x0014      count 36
msg 0x0047      count 36
msg 0x0005      count 36
msg 0x000f      count 36
msg 0x000d      count 1
```

Reply:
Per Discord, keeping it simple :)